core/templates/pages/exploration-editor-page/improvements-tab/services/improvement-modal.service.ts

```diff
@@ -324,7 +324,7 @@ angular.module('oppia').factory('ImprovementModalService', [
 
               // TODO(Allan): Implement ability to edit suggestions before
               // applying.
-              $scope.addNewMessageAsync = function(
+              $scope.addNewMessage = function(
```

Comment:
Seems like this still calls an async function -- could you explain the rationale behind removing "async"?

(Ditto elsewhere. Not saying it's incorrect, but just want to make sure we have a consistent standard.)

Reply:
Ah, that's because the frontend is using it as button functions. I don't find the "async-ness" of the implementation good enough of a reason to complicate the directive, so I left it out.

Reply:
OK. In that case, what should our general policy be for use of that suffix? If you could propose one, let's then hash it out and add it to the coding style guide.

Reply:
I think directive-functions should be the exception to the rule, but otherwise Async in function names sounds good to me